org.gimp.GIMP.json

```diff
@@ -314,6 +314,9 @@
                 {
                     "name": "exiv2",
                     "buildsystem": "cmake-ninja",
+                    "config-opts": [
+                        "-DEXIV2_ENABLE_BMFF=ON"
```

Comment:
I'm sure you must have said it somewhere else on some comment, and I can see in your commit on GIMP repo, you said:

> Improves AVIF, HEIF, JPEG XL metadata support.

What does it improve exactly, regarding metadata?

Reply:
Without that switch, exiv2 do not support these new formats.
That switch will be default in the future releases anyway, just in the past some people were worried if some patent trolls might claim they invented copying of data from one array to another.